### PR TITLE
feat: introduce z index prop to date range picker

### PR DESCRIPTION
**Pull request by AlanRad-QH**
## Description

Additional `zIndex` for `DateRangePicker` component to fix stack order

💁‍

## Related issues

[](https://app.shortcut.com/qantasaccommodation/story/127565/fix-info-icon-tooltip-showing-on-top-of-calendar)

💪

## Looks like this

Screen Shot 2023-06-21 at 10 48 32 am
Screen Shot 2023-06-21 at 10 48 58 am

🤨

---

**PR check-list**

🚨 **IMPORTANT:** Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository first! 👀

- [ ] You've checked if you need to update any relevant typescript declaration file(s), for example component props (including styled-system props) or theme.
- [ ] You've requested a review from at least ONE person. Preferably a roo-ui maintainer ([Philip](https://github.com/philipwindeyer), [Eric](https://github.com/eneo5541), [Mike](https://github.com/memcc), [Marcel](https://github.com/Anthropic), [Sam](https://github.com/SamuSan), [Long](https://github.com/KieraDOG)) or a Roo UI working-group human.
  - Note: If your PR has not been reviewed within two days of requesting a review, feel free to seek reviews from other [team members of Qantas Hotels](https://github.com/orgs/hooroo/people).
- [ ] Your final merge commit message is in the format `<type>(<scope>): <description>` as per [the contributing guide](https://github.com/hooroo/roo-ui/blob/master/.github/CONTRIBUTING.md#commit-naming)
  - Example: `fix(dependencies): minor version bump to resolve security vulnerability in set-value`

### CODE REVIEW

**Review by apalski — APPROVED** [review]
LGTM 👍

File: src/components/DatePicker/DateRangePicker.js [hunk 1]
```diff
@@ -222,6 +227,7 @@ DateRangePicker.defaultProps = {
   isSettingEndDate: false,
   onRangeSelected: NOOP,
   onChangeDates: NOOP,
+  zIndex: 1,
```
Comment (ajlu) [inline src/components/DatePicker/DateRangePicker.js:230]: should there be no new default to ensure backwards compatibility?

**Review by ajlu — APPROVED** [review]

File: src/components/DatePicker/DateRangePicker.js (hunk 1 above)
Comment (AlanRad-QH) [reply src/components/DatePicker/DateRangePicker.js]: Updated. Thanks.

**Pull request merged**